### ISSUE TITLE
Brave icon missing on Private Tab

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-ios#1672]
### Description:
Brave icon missing from Private Tab

### Steps to Reproduce

  1. Launch 1.6.3 (18.07.11.14)
  2. Open a Private tab.
  3. Answer modals.

**Actual result:**
Once modals have cleared, you will see that the Brave icon is missing. 
![img_0214](https://user-images.githubusercontent.com/28145373/42709879-fcfb3a34-86af-11e8-990a-18466b77644d.PNG)

**Expected result:**
Brave icon should still be on the Private Tab page as it is in app store version:
![img_0215](https://user-images.githubusercontent.com/28145373/42709888-072c5a4c-86b0-11e8-9160-d7f4c9f43891.PNG)

**Reproduces how often:** [Easily reproduced/Intermittent issue]
Easily

**Brave Version:**
1.6.3 (18.07.11.14)

**Device details:**
Found with iPad on iOS10

**Website problems only:**
- did you check with Brave Shields down?

- did you check in Puffin/Dolphin (UIWebView-based browsers)? 

- did you check in Safari/Firefox (WkWebView-based browsers)? 

### Additional Information
Since Private Tab contains additional information if you don't turn on DDG from the modal, should probably check this when it's fixed with DDG selected and DDG not selected for Private Tabs.

**Comment by iccub** [comment]
This is a design decision as far as I'm aware. Showing the Brave logo only if ddg promo label is not present seems weird for me. Will talk to @jamesmudgett about it after he's back from vacation.

**Comment by LaurenWags** [comment]
@iccub are you saying that the Brave logo should be displayed only if DDG promo label is present? because that's not the case either:

![img_0219](https://user-images.githubusercontent.com/28145373/42822034-fd8df2f2-89a7-11e8-989c-0b4442329cb0.PNG)

**Comment by iccub** [comment]
Sorry for not being clear, the logo got completely removed from code, and I think it was done on purpose.
But yeah, especially on iPad, looks like there's still some space to have it.

Commit for full logo removal was pushed last month, along with the DDG promo https://github.com/brave/browser-ios/commit/d70c6960b277326e55eef3e8db1856acbff90a73#diff-abfc8d88f8fbb974154bce59e22c81a6

**Comment by jhreis** [comment]
Yeah, this does look intentional. We should confirm with design team though before pushing this live.

**Comment by LaurenWags** [comment]
Icon on iOS10 iPad using 1.6.3 (18.07.18.11):
![img_0220](https://user-images.githubusercontent.com/28145373/42905079-fbd50a42-8aa4-11e8-9862-f585fc95afb5.PNG)
